src/jmc/compile/command/builtin_function/bool_function.py

```diff
@@ -25,6 +25,19 @@ def call_bool(self) -> tuple[str, bool, list[str]]:
         return f'score {self.args["selector"]} {self.args["objective"]} matches 1..', UNLESS, [
         ]
 
+# please put this somewhere in appropriate place, idk where to put these 2 functions so for now they're here.
+class nbtSource: 
+    def is_uuid(string) -> str:
+        parts = string.split('-')
+        return len(parts) == 5 and all(len(part) in (8, 4, 4, 4, 12) and part.isalnum() for part in parts)
+    def get_source_type(source) -> str:
+        if source.startswith("@") or ObjectIsEqual.is_uuid(source):
+            source_type = "entity"
+        elif source[0] in "~^" or source.isnumeric():
+            source_type = "block"
+        else:
+            source_type = "storage"
+        return source_type
```

Comment:
Unnecessary value for returning
Guard Clause is preferred

Reply:
that has to be it, i think i fixed it.